src/bn256/fq2.rs

```diff
@@ -271,12 +272,14 @@ impl Fq2 {
     }
 
     pub fn frobenius_map(&mut self, power: usize) {
-        self.c1 *= &FROBENIUS_COEFF_FQ2_C1[power % 2];
+        if power % 2 != 0 {
+            self.conjugate()
```

Comment:
Conjugate can't be a `const fn` as it is done "in place". I wonder if we can gain any speedup by not making it get an `&mut` and declaring the fn `const`.

kk since it relies in `Sub` which isn't const this will not work. So this is fine as is.
Just leaving the comment with my reasoning so you can double-check